### ISSUE TITLE
ERROR __main__ Error tokenizing data. C error: EOF inside string starting at row 473

### ISSUE THREAD

**Issue by zhangqiqi1228** [UBIAI/layoutlmv3FineTuning#2]
Hi, thanks for your scripts. 

I finetuned the "microsoft/layoutlmv3-base" with my customized dataset (5 labels). Then, I used the finetuned model to run inference on some PNG files, which have the same size and format as the training dataset. 

However, when I ran the run_inference.py, there was an error: ERROR __main__ Error tokenizing data. C error: EOF inside string starting at row 473. 

I searched for the explanations related with this error. It seems to be something wrong with loading the TSV file. So, I checked the TSV file separately, but it worked and I successfully printed the word_text, which contains the words and boxes. 

Do you know what causes this error? Where is the row 473 mentioned in the error? Thanks.
